Tasks/AppCenterDistributeV3/Tests/UnitTests/TestHelpers.ts

```diff
@@ -0,0 +1,18 @@
+import * as assert from "assert";
+
+/**
+ * Exit code is used to determine whether unit test passed or not.
+ * When executing code requires vsts-task-lib somewhere it makes exit code = 0 regardless whether exception was thrown.
```

Comment:
> When executing code requires vsts-task-lib somewhere it makes exit code = 0 regardless whether exception was thrown.

Is it possible to fix/extend `vsts-task-lib` to not perform the `exit code = 0` step, rather than write tests outside of this via `spawnSync`?

Reply:
Additionally, would it be valuable to write a unit test where we expect `exit code = 1` (i.e., an unhappy path unit-test) to ensure that our child processes aren't also having their exit code overwritten?

Reply:
We've add "negative path" test to check that child process exit code is kept.
But what about changing `vsts-task-lib` behaviour:
- it intended for integration testing of tasks, but we need unit test for our code
- also changing that lib behaviour would affect entire `azure-pipeline-tasks` repo and reviewing process could take more time while we are trying to go with current release train so let's stuck with current approach 🙂